koji-containerbuild.spec

```diff
@@ -63,7 +63,8 @@ Requires:   python3-six
 %description builder
 Builder plugin that extend Koji to communicate with OpenShift build system and
 build layered container images.
-%endif # with_python3
+%endif
+# with_python3
```

Comment:
```suggestion
# end with_python3
%endif
```

and you know my opinion about unrelated changes in the same commit :)

Reply:
changed and added in separate commit